### PR TITLE
Kev branch

### PR DESCRIPTION
**Pull request by FeliciaNemet**
Hey Team! Are we ready to merge Kevin's changes into the devBranch?

### CODE REVIEW

File: src/components/Dashboard.js [hunk 1]
```diff
@@ -62,8 +63,25 @@ const Dashboard = () => {
...
           />
+          <Route
+            path="MarketPlace/*"
+            element={
+              <div>
+                <h1>Free Market</h1>
+                <MarketPlaceInputs />
+              </div>
+            }
+          />
```
Comment (vjoseph3) [inline src/components/Dashboard.js:74]: Looks like lines 57-65 are accidentally duplicated on lines 66-74

File: src/components/UserInputs.js [hunk 2]
```diff
@@ -21,9 +21,7 @@ const UserInputs = () => {
           console.log(`hello`);
           navigate('/Dashboard',
           
-     )
-
-     }
+     )}
```
Comment (vjoseph3) [inline src/components/UserInputs.js]: While we're fixing the formatting here, why not replace lines 22-26 (now 22-24) with the following?
```
          navigate('/Dashboard');
     }
```

File: src/components/marketplace/nirvanaPages/RequestPayment.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+            <input type="text" name="To" required />
+          </div>
+          <div className="amountBox">
+            <label htmlFor="Amount">Amount:</label>
+            <input type="text" name="Amount" required />
+          </div>
+          <div className="amountBox">
+            <label htmlFor="Amount">Amount:</label>
+            <input type="text" name="Amount" required />
+          </div>
```
Comment (vjoseph3) [inline src/components/marketplace/nirvanaPages/RequestPayment.js]: Is the duplicate Amount field here intentional? Also in MakePayment.js

**Comment by vjoseph3** [comment]
This looks great to me, nice work Kevin. This is really taking shape!

Hope you don't mind my business-like tone noting a couple harmless copy-paste artifacts in the code. I think a simple commit and push would add any tweaks directly to this PR.

It's motivating as always to see this progress. Thanks for your hard work and Felicia for the PR!

File: src/components/Dashboard.js (hunk 1 above)
Comment (FeliciaNemet) [reply src/components/Dashboard.js:74]: Kevin and I are meeting up for 3:30ish today to work on the component, I will make sure this is fixed :) You're welcome to join us if you'd like!

**Pull request merged**